### PR TITLE
Changelog and readme update for 11.3-a.5

### PR DESCRIPTION
**Pull request by bindlegirl**
This is update for the changelogs and readme file for Jetpack.

#### Changes proposed in this Pull Request:

* Updated files

#### Other information:

- [ ] Have you written new tests for your changes, if applicable?
- [ ] Have you checked the E2E test CI results, and verified that your changes do not break them?

#### Jetpack product discussion

n/a

#### Does this pull request change what data or activity we track or use?

no

#### Testing instructions:

* Check files
* Please make sure you read the Jetpack changelog and readme as this should be the final version

### CODE REVIEW

File: projects/plugins/jetpack/CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,27 @@
...
 
+## 11.3-a.5 - 2022-08-16
+### Enhancements
+- Payments block: add a new `useWidth` hook to control the width settings of a block. [#25394]
+- Google Analytics: add support for the DNT sent by the browser. [#25423]
+- Dashboard: add new card to highlight options available to WordPress agencies. [#25041]
+
+### Improved compatibility
+- VideoPress: migrate code from the plugin to the package. [#25412]
+- VideoPress: move dependencies to the package. [#25398]
```
Comment (coder-karen) [inline projects/plugins/jetpack/CHANGELOG.md]: Both of these seem more behind the scenes changes, would fit better under other changes I think.

File: projects/plugins/jetpack/readme.txt [hunk 2]
```diff
@@ -242,16 +242,22 @@ Jetpack Backup can do a full website migration to a new host, migrate theme file
...
-- Payments block: use Block API v2 to simplify the overall markup.
-- Payments block: ensure the plan name field will update according to the other options selected, unless it's already been modified.
+- Payments block: add a new `useWidth` hook to control the width settings of a block.
+- Google Analytics: add support for the DNT sent by the browser.
+- Dashboard: add new card to highlight options available to WordPress agencies.
 
 #### Improved compatibility
-- Backup: add disclaimer text and link to the backup product card that links to an FAQ on the Pricing page.
+- VideoPress: migrate code from the plugin to the package.
+- VideoPress: move dependencies to the package.
```
Comment (coder-karen) [inline projects/plugins/jetpack/readme.txt]: Same as with the changelog, though as there is no 'other' here they could just go.

**Review by coder-karen — CHANGES_REQUESTED** [review]

File: projects/plugins/jetpack/readme.txt (hunk 2 above)
Comment (bindlegirl) [reply projects/plugins/jetpack/readme.txt]: Resolved in [`62f9de3`](https://github.com/Automattic/jetpack/pull/25649/commits/62f9de3dbcf99df3c6538b28c624203120ad3966)

**Review by coder-karen — APPROVED** [review]

**Comment by bindlegirl** [comment]
r250727-wpcom

**Pull request merged**